test/StakeManager.t.sol

```diff
@@ -144,6 +144,53 @@ contract UnstakeTest is StakeManagerTest {
         assertEq(ERC20(stakeToken).balanceOf(address(userVault)), 0);
         assertEq(ERC20(stakeToken).balanceOf(testUser), 1000);
     }
+
+    function test_UnstakeShouldBurnMultiplierPoints() public {
+        uint256 stakeAmount = 1000;
+        uint256 percentToBurn = 90;
+        deal(stakeToken, testUser, stakeAmount);
+        StakeVault userVault = _createTestVault(testUser);
+
+        vm.startPrank(testUser);
+        ERC20(stakeToken).approve(address(userVault), stakeAmount);
+
+        userVault.stake(stakeAmount, 0);
+
+        assertEq(stakeManager.totalSupplyMP(), stakeAmount);
+        for (uint256 i = 0; i < 53; i++) {
```

Comment:
Just cosmetics, but `53` here is somewhat of an arbitrary number. Would be better to give some semantic meaning using a variable or constant or something like that.

Reply:
53 is to complete a year. 53 weeks gives roughly 1 year.